### PR TITLE
Fixed Registered Events tab by adding event title from GraphQL

### PR DESCRIPTION
**Pull request by ayushs136**
**What kind of change does this PR introduce?**
Fixed Registered Events tab by displaying registered events from GraphQL

From 
image

To 
image

### CODE REVIEW

**Comment by Sagar2366** [comment]
resolve conflicts in PR

File: lib/views/pages/login_signup/register_form.dart [hunk 1]
```diff
@@ -286,9 +286,10 @@ class RegisterFormState extends State<RegisterForm> {
                           prefixIcon: Icon(Icons.lock, color: Colors.white),
                           suffixIcon: FlatButton(
                             onPressed: _toggle,
-                            child: Icon(_obscureText
-                                ? Icons.visibility_off
-                                : Icons.visibility,
+                            child: Icon(
```
Comment (Sagar2366) [inline lib/views/pages/login_signup/register_form.dart:289]: Please, do not add any changes other than part of the issue.
Revert all these changes to merge this PR.

**Comment by Sagar2366** [comment]
Closing this issue as it is already fixed under https://github.com/PalisadoesFoundation/talawa/pull/283

**Comment by ayushs136** [comment]
> Closing this PR as it is already fixed under https://github.com/PalisadoesFoundation/talawa/pull/283

In #283, the fix is only if user isn't registered in any event, but that does not displays the registered events list. But this PR fix that.

**Pull request closed without merging**